tensorflow/contrib/makefile/Makefile

```diff
@@ -687,6 +801,18 @@ $(PROTO_OBJS): $(PROTO_CC_SRCS) $(PBT_GEN_FILES)
 
 # Host compilation rules.
 
+# NVCC compilation rules for Tegra
+ifeq ($(BUILD_FOR_TEGRA),1)
+$(OBJDIR)%.cu.o: %.cu.cc
```

Comment:
Would it be possible for you to verify this rule works with Make 3.81? I'm seeing that it's not getting matched and the build fails when it tries to use g++ to compile cu.cc files. I've tried moving it up and down in the rule list as well to see if it's an order thing, but it seems to make no difference.

Reply:
Make 3.81 was released in 2006.
I can't find a working binary and I had a lot of problem building from source.
However, can you try moving the

ifeq ($(BUILD_FOR_TEGRA),1)
$(OBJDIR)%.cu.o: %.cu.cc
	@mkdir -p $(dir $@)
	@mkdir -p $(dir $(DEPDIR)$*)
	$(NVCC) $(NVCCFLAGS) -Xcompiler "$(CXXFLAGS4NVCC) $(DEPFLAGS)" $(INCLUDES) -c $< -o $@

$(LIBDIR)libtfcuda.a: $(CUDA_CC_OBJS)
	@mkdir -p $(dir $@)
	$(AR) $(ARFLAGS) $@ $(CUDA_CC_OBJS)
endif


above

$(OBJDIR)%.o: %.cc | $(PBT_GEN_FILES)
	@mkdir -p $(dir $@)
	@mkdir -p $(dir $(DEPDIR)$*)
	$(CXX) $(CXXFLAGS) $(DEPFLAGS) $(INCLUDES) -c $< -o $@
	@mv -f $(DEPDIR)/$*.Td $(DEPDIR)/$*.d

It should work theoretically.

Reply:
I tested on my machine. With above change, I can compile with GNU make 3.81